cos-fleetshard-operator-camel/src/main/java/org/bf2/cos/fleetshard/operator/camel/CamelOperandSupport.java

```diff
@@ -340,6 +335,40 @@ public static Map<String, String> createSecretsData(
                 cfg.exchangePooling().exchangeFactoryStatisticsEnabled());
         }
 
+        // configure the empty config map created for logging
+        final ConfigMap configMap = connectorConfiguration.getConfigMap();
```

Comment:
if I properly understand this block, we take data from the configmap and copy it over the secret that already exists but another option would be to mount the configmap and let the runtime load any property that is defined (so no need to translate `log.level` into specific quarkus properties (which, as side effect, would allow to also add specific properties to tweak the behavior  of the connector if needed)

Reply:
I recall part of this being discussed and one of the points was how much control we want to have over it. If we do that, we lose control entirely.
Also, if we do go down that route, I'd question many of the current exposed/configured properties in connectors or even operators, since we could just put everything in the configmaps and expose all of it.

Reply:
@valdar wdyt?

Reply:
> I recall part of this being discussed and one of the points was how much control we want to have over it. If we do that, we lose control entirely. Also, if we do go down that route, I'd question many of the current exposed/configured properties in connectors or even operators, since we could just put everything in the configmaps and expose all of it.

Ok so the implemented behavior is fine with me.

The question was mainly related to how the same config map approach works for the sync & operators where we don't have any kind of control whereas here we decided to go into a more controlled way.

Reply:
@lburgazzoli @rinaldodev I think that the idea behind the 2 different approaches was based on how often we foreseen to use them. The idea is that connectors are the more  likely to be placed in debug mode and so the one for which we want to prevent erroneous configuration the most.

That said, the counter argument is that something is not working if we need to activate debug mode, so as long as the correct values are documented in SOPs we should be good and gain flexibility for future unforeseen needs. Also there is the argument of consistency across system components...

Reply:
> Also, if we do go down that route, I'd question many of the current exposed/configured properties in connectors or even operators, since we could just put everything in the configmaps and expose all of it.

I am not sure I completely get you: the fact that there is the possibility of overriding properties putting them in a configmap dose not make the current way of using some properties less effective, I think.

Reply:
@valdar it doesn't make them less effective, I'd just opt for a single one in favor of simplicity.

I understand we can then move forward with the current solution.

Reply:
@valdar and I have discussed this and opted for a more flexible solution.